lxd/networks.go

```diff
@@ -1488,11 +1488,26 @@ func networkStateGet(d *Daemon, r *http.Request) response.Response {
 		return resp
 	}
 
-	name := mux.Vars(r)["name"]
+	networkName := mux.Vars(r)["name"]
 
-	state, err := resources.GetNetworkState(name)
-	if err != nil {
-		return response.SmartError(err)
+	projectName := queryParam(r, "project")
```

Comment:
@markylaing any reason you went with this approach rather than using `projectParam()` which houses the same logic?

Reply:
Ah yeah, good catch, we should probably just use the standard function for that.

Reply:
No reason at all I just forgot that the function existed :grimacing: 

How would I go about changing this now that it's merged? Or is it just a case of editing the commit and pushing to `lxc:master` (for someone with sufficient perms)?

Reply:
Just fix it in a separate commit in your other PR or in a new PR of its own.

We never edit git history as that would break everyone who has a clone of the repo.